packages/salesforcedx-utils-vscode/test/unit/telemetry/MockContext.ts

```diff
@@ -26,10 +32,45 @@ class MockMemento implements Memento {
   }
 }
 
+class MockEnvironmentVariableCollection
```

Comment:
Nothing inside of this Mock class is directly used so we can get away with not implementing all the logic.